### PR TITLE
Y'all n'wahs aint even smoke - adds skooma

### PR DESCRIPTION
**Pull request by deathride58**
Title. Skooma is a drug from the elder scrolls games, a highly addictive drug that greatly increases the user's speed and strength, but ruins their dexterity/agility and intelligence. This works within the limitations of this codebase to try to emulate it, and also tries to emulate the withdrawal effects shown by various NPCs in the TES games.

To make skooma, you first have to make moon sugar. Moon sugar can be made by slapping moonshine and sugar together into a beaker in a 1:1 ratio, then slapping it into a reagent grinder to mix the reagents.
After you've made moon sugar, you can then make skooma by combining 10u of moon sugar with 5u of morphine inside of a container chilled below room temperature, which will produce 2u of skooma, 4u of moonshine, and 4u of sugar. Heating the resulting skooma above normal body temperature will result in the 2u moonsugar and 5u of morphine spent in the recipe returning (even inside of bodies, do as you will with that info). This tedious recipe ensures that cases where someone inhales 1000u of skooma are insanely rare and well-deserved whenever they end up happening.
Skooma itself causes addictions at a minimum of 1u, but it's impossible to actually overdose on it, meaning you can fit as much skooma inside you as you can make.
When it's in your system, skooma increases your movement speed, halves all incoming stamina adjustments, multiplies your maximum unarmed punch damage by 5, and doubles your click delay. Every tick, it has a small chance to make your eyes blurry for two deciseconds, and deals minor brain damage and minor toxinloss, 1u of skooma = 5 brain damage and 5 toxinloss. It's definitely best to mix it with other chems to counteract the harmful side effects.
Skooma's withdrawal lasts for a total of approximately 43 minutes, the majority of it being the final stage. The withdrawal starts off with jitteriness and eye blurriness, with dizziness being introduced during the second stage. The third stage lasts twice as long as the first two stages, and introduces involuntary emotes. The fourth final stage has much more intense variants of the above effects, and lasts approximately 40 minutes, dealing potentially lethal brain damage as it continues. It's entirely plausible to tough out the withdrawal process with some help from chemistry, but the game will be constantly tempting you back towards the feline devil's nectar.

:cl: deathride58
add: The Khajiit favorite, Skooma, was added as a chemistry recipe that can be produced with cooperation between chemistry and either cargo or service. If you want to spoil the recipe for yourself, take a look at the PR for it!
/:cl:

### CODE REVIEW

File: code/modules/reagents/chemistry/machinery/reagentgrinder.dm [hunk 1]
```diff
@@ -301,7 +301,7 @@
 	operate_for(50, juicing = TRUE)
 	addtimer(CALLBACK(src, /obj/machinery/reagentgrinder/proc/mix_complete), 50)
 
-/obj/machinery/reagentgrinder/proc/mix_complete()
+/obj/machinery/reagentgrinder/proc/mix_complete() //what the fuck. why the fuck are mixed recipes hardcoded. seriously what in the actual shit. i mean its out of scope for my pr but hot damn
```
Comment (Poojawa) [inline code/modules/reagents/chemistry/machinery/reagentgrinder.dm]: tbh, unneeded rant comment

Comment (deathride58) [reply code/modules/reagents/chemistry/machinery/reagentgrinder.dm]: but MOOOOOOOOOOOM, how else am i gonna squeeze as many line of code changes out of my PRs?

Comment (nik707) [reply code/modules/reagents/chemistry/machinery/reagentgrinder.dm]: i actually am ok with this comment thank you for ur opinions papa robo

**Review by nik707 — APPROVED** [review]
this is honestly better than the med rework.

**Pull request merged**